app/V1Module/router/RouterFactory.php

```diff
@@ -331,6 +331,7 @@ private static function createUsersRoutes(string $prefix): RouteList {
     $router[] = new PostRoute("$prefix/<id>", "Users:updateProfile");
     $router[] = new PostRoute("$prefix/<id>/settings", "Users:updateSettings");
     $router[] = new PostRoute("$prefix/<id>/create-local", "Users:createLocalAccount");
+    $router[] = new PostRoute("$prefix/<id>/role", "Users:setRole");
```

Comment:
So this was implemented before and we didn't make a route for it?

Reply:
AFAIK, yes.

Reply:
Yes... this is my fault, I forgot the route.

Reply:
No hard feelings, just curiosity :)